Test/COD51010.TXT

```diff
@@ -0,0 +1,218 @@
+OBJECT Codeunit 51010 Test_DotNet_SymmetricAlgorithm
+{
+  OBJECT-PROPERTIES
+  {
+    Date=11/13/18;
+    Time=[ 4:00:25 PM];
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    Subtype=Test;
+    TestPermissions=Disabled;
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      Assert@1001 : Codeunit 130000;
+      IsInitialized@1000000002 : Boolean;
+      Algorithm@1000000000 : 'Aes,DES,RC2,Rijndael,TripleDES';
+      Mode@1000000003 : ',CBC,ECB,OFB,CFB,CTS';
+      Padding@1000000001 : ',None,PKCS7,Zeros,ANSIX923,ISO10126';
+
+    LOCAL PROCEDURE Initialize@1000000005();
```

Comment:
This isn't really needed 😃

Reply:
Please elaborate -- do you want me to remove the Initialize() function (since it does nothing)?

Reply:
Yes, do remove it.

Somethime when initialization is needed once per codeunit, it make sense to have such a function. But in this case it's useless.